src/devices/team/nm-device-team.c

```diff
@@ -479,11 +480,25 @@ teamd_child_setup (gpointer user_data)
 	signal (SIGPIPE, SIG_IGN);
 }
 
+static const char **
+teamd_env (void)
+{
+	const char **env = g_new0 (const char *, 2);
+
+	if (nm_config_get_is_debug (nm_config_get ()))
+		env[0] = "TEAM_LOG_OUTPUT=stderr";
+	else
+		env[0] = "TEAM_LOG_OUTPUT=syslog";
```

Comment:
does teamd need no environment variables at all? I wonder if there are some important once that we need to pass on as well?

Reply:
It doesn't need any environment variable; indeed at the moment when NetworkManager is spawned by systemd (and teamd inherits the environment) the only variables set are PATH, LANG and the various LC_*, which are not used by teamd.

Reply:
ACK